tests/js/arraydemo.scrypttest.js

```diff
@@ -41,27 +101,46 @@ describe('Test sCrypt contract ArrayDemo In Javascript', () => {
     });
 
 
-    // it('should return true', () => {
-    //     result = arraydemo.testArrayInt([1, 1321, 243213, 32132]).verify()
-    //     expect(result.success, result.error).to.be.true
-    // });
+    it('should return true', () => {
```

Comment:
Some public functions are not tested, such as `unlock()`. Let's cover all.

Reply:
add